src/node_options.h

```diff
@@ -400,6 +401,7 @@ class OptionsParser {
   friend class OptionsParser;
 
   friend void GetOptions(const v8::FunctionCallbackInfo<v8::Value>& args);
+  friend std::string GetBashCompletion();
```

Comment:
This could also be a member of `OptionsParser`, right?

Reply:
It can, but so is `GetOptions()` so I figured it's better to keep them consistent